typescript/src/lib/execution/kurtosis_lambda_executor.ts

```diff
@@ -0,0 +1,84 @@
+import { ApiContainerServiceClient, NetworkContext } from "kurtosis-core-api-lib";
+import { err, ok, Result } from "neverthrow";
+import { API_CONTAINER_SOCKET_ENV_VAR, EXECUTION_VOLUME_MOUNTPOINT, SERIALIZED_CUSTOM_PARAMS_ENV_VAR } from "../../kurtosis_lambda_docker_api/kurtosis_lambda_docker_api";
+import { KurtosisLambda } from "../kurtosis_lambda/kurtosis_lambda";
+import { KurtosisLambdaConfigurator } from "./kurtosis_lambda_configurator";
+import * as grpc from 'grpc';
+import { LambdaServiceService } from "../../kurtosis_lambda_rpc_api_bindings/api_lambda_service_grpc_pb";
+import { MinimalGRPCServer, TypedServerOverride } from "minimal-grpc-server";
+import { LISTEN_PORT } from "../../kurtosis_lambda_rpc_api_consts/kurtosis_lambda_rpc_api_consts";
+import { KurtosisLambdaServiceServer } from "./kurtosis_lambda_service_server";
+
+const GRPC_SERVER_STOP_GRACE_PERIOD_SECONDS: number = 5;
+
+export class KurtosisLambdaExecutor {
+    private readonly configurator: KurtosisLambdaConfigurator;
+
+    constructor(configurator: KurtosisLambdaConfigurator) {
+        this.configurator = configurator;
+    }
+
+    async run(): Promise<Result<null, Error>> {
+        const getSerializedCustomParamsResult: Result<string, Error> = KurtosisLambdaExecutor.getEnvVar(SERIALIZED_CUSTOM_PARAMS_ENV_VAR, "the serialized custom params that the Lambda will consume");
+        if (getSerializedCustomParamsResult.isErr()) {
```

Comment:
Interesting, I write my cases like (! getSerializedCustomParamsResult.isOk() ) instead. It is the same thing, but I thought that out.

Reply:
yep! I don't think there's any difference - six of one, half-dozen of the other. The only reason I prefer `.isErr()` is because then I don't have to remember the `!` (so less bug opportunity)